napari/layers/_image_layer/model.py

```diff
@@ -326,6 +332,33 @@ def interpolation_functions(self):
     def _clim_range_default(self):
         return [float(self.image.min()), float(self.image.max())]
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current image data and colormap.
+        """
+        zoom_factor = np.divide(self._thumbnail_shape[:2],
+                                self._image_view.shape[:2]).min()
+        if self.multichannel:
+            downsampled = ndi.zoom(self._image_view,
+                                   (zoom_factor, zoom_factor, 1),
+                                   prefilter=False, order=0)
+            if self._image_view.shape[2] == 4:
+                mapped = downsampled
+            else:
+                ones = 255 * np.ones(list(downsampled.shape[:2]) + [1])
```

Comment:
```suggestion
                ones = np.full(downsampled.shape[:-1] + (1,), 255, dtype=np.uint8)
```

Reply:
also don't think that `ones` is a good name for this since it doesn't actually consist of ones

Reply:
fixed and renamed alpha